### ISSUE TITLE
`clocker data --gt=...` wrong for entries over midnight

### ISSUE THREAD

**Issue by fnogatz** [substack/clocker#28]
![screenshot from 2015-09-14 08 47 14](https://cloud.githubusercontent.com/assets/881988/9843744/6919178a-5abd-11e5-80da-7676348e7e75.png)

**Comment by fnogatz** [comment]
The same applies for `clocker data` without the `--gt` flag:

```sh
> clocker -d /tmp/clocker start -t some
> clocker -d /tmp/clocker set start 'yesterday 23:00'
> clocker -d /tmp/clocker stop                       
> clocker -d /tmp/clocker set end '01:00'    
> clocker -d /tmp/clocker list
1534798800  2018-08-20  [ 23:00:00 - 01:00:00 ]  (02:00:00)  [some]
> clocker -d /tmp/clocker data
[
  {
    "hours": [
      {
        "date": "2018-08-21",
        "hours": 2
      }
    ],
    "title": "consulting"
  }
]
```

This should instead return objects for `2018-08-20` and `2018-08-21` with 1 hour each.

**Comment by fnogatz** [comment]
This is fixed since #40 in `v1.15.0`.
